### PR TITLE
Patch 1- public-s3-cloudfront-origin

### PR DESCRIPTION
**Pull request by shuklaalok87**

### CODE REVIEW

**Comment by shuklaalok87** [comment]
Secure Origin Protocol

File: en/aws/cloudfront/cloudfront-logging-enabled.md [hunk 1]
```diff
@@ -15,7 +15,7 @@
 | **Recommended Action** | Enable CloudFront request logging. |
 
 ## Detailed Remediation Steps
-1. Log into the AWS Management Console.
+1. Log into to the AWS Management Console.
```
Comment (matthewdfuller) [inline en/aws/cloudfront/cloudfront-logging-enabled.md]: This shouldn't change here

File: resources/aws/cloudtrail/cloudtrail-bucket-private/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (matthewdfuller) [inline resources/aws/cloudtrail/cloudtrail-bucket-private/README.md]: This file shouldn't be needed

File: resources/aws/cloudtrail/cloudtrail-enabled/README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (matthewdfuller) [inline resources/aws/cloudtrail/cloudtrail-enabled/README.md]: This file shouldn't be needed

File: resources/aws/cloudtrail/cloudtrail-bucket-delete-policy/README.md [hunk 4]
```diff
@@ -0,0 +1 @@
+
```
Comment (matthewdfuller) [inline resources/aws/cloudtrail/cloudtrail-bucket-delete-policy/README.md]: This file shouldn't be needed

**Pull request merged**